rust-runtime/aws-smithy-http-server/Cargo.toml

```diff
@@ -1,14 +1,14 @@
 [package]
 name = "aws-smithy-http-server"
 version = "0.1.0"
-authors = ["Smithy Rust Server <smithy-rs-server@amazon.com>"]
+authors = ["Matteo Bigoi <matbigoi@amazon.com>", "David Pérez <davidpz@amazon.com>"]
```

Comment:
Please leave the alias instead of names. It will make things easier in the future as the email list is already active and contains me and you right now.